.circleci/config.yml

```diff
@@ -0,0 +1,200 @@
+            libfreetype6-dev
+      - run: |
+          echo LANGUAGE: $LANGUAGE
+          echo LANG: $LANG
+          echo LC_MESSAGES: $LC_MESSAGES
+          echo LC_ALL: $LC_ALL
+      - run: ./buildconf -f
+      - run: |
+          ./configure \
+            --enable-debug \
+            --enable-zts \
+            --enable-option-checking=fatal \
+            --prefix=/usr \
+            --enable-phpdbg \
+            --enable-fpm \
+            --enable-opcache \
+            --with-pdo-mysql=mysqlnd \
+            --with-mysqli=mysqlnd \
+            --with-pgsql \
+            --with-pdo-pgsql \
+            --with-pdo-sqlite \
+            --enable-intl \
+            --without-pear \
+            --enable-gd \
+            --with-jpeg \
+            --with-webp \
+            --with-freetype \
+            --with-xpm \
+            --enable-exif \
+            --with-zip \
+            --with-zlib \
+            --with-zlib-dir=/usr \
+            --enable-soap \
+            --enable-xmlreader \
+            --with-xsl \
+            --with-tidy \
+            --enable-sysvsem \
+            --enable-sysvshm \
+            --enable-shmop \
+            --enable-pcntl \
+            --with-readline \
+            --enable-mbstring \
+            --with-curl \
+            --with-gettext \
+            --enable-sockets \
+            --with-bz2 \
+            --with-openssl \
+            --with-gmp \
+            --enable-bcmath \
+            --enable-calendar \
+            --enable-ftp \
+            --with-pspell=/usr \
+            --with-enchant=/usr \
+            --with-kerberos \
+            --enable-sysvmsg \
+            --with-ffi \
+            --enable-zend-test \
+            --enable-dl-test=shared \
+            --with-ldap \
+            --with-ldap-sasl \
+            --with-password-argon2 \
+            --with-mhash \
+            --with-sodium \
+            --enable-dba \
+            --with-cdb \
+            --enable-flatfile \
+            --enable-inifile \
+            --with-tcadb \
+            --with-lmdb \
+            --with-qdbm \
+            --with-snmp \
+            `#--with-unixODBC` \
+            --with-imap \
+            --with-kerberos \
+            --with-imap-ssl \
+            `#--with-pdo-odbc=unixODBC,/usr` \
+            `#--with-pdo-oci=shared,instantclient,/opt/oracle/instantclient` \
+            `#--with-oci8=shared,instantclient,/opt/oracle/instantclient` \
+            --with-config-file-path=/etc \
+            --with-config-file-scan-dir=/etc/php.d \
+            --with-pdo-firebird \
+            `#--with-pdo-dblib` \
+            --disable-phpdbg \
+            `#--enable-werror`
+      - run: make -j$(/usr/bin/nproc) > /dev/null
+      - run: sudo make install
+      - run: sudo mkdir -p /etc/php.d
+      - run: sudo chmod 777 /etc/php.d
+      - run: echo opcache.enable_cli=1 > /etc/php.d/opcache.ini
+      - run: echo opcache.protect_memory=1 >> /etc/php.d/opcache.ini
```

Comment:
You can use multiple command in a single run which is usually better to group things together. Also it's good to use `name` for run and `command` for the command.

Reply:
Right, I split it because in case of failure it's non-trivial to see which line failed. I don't mind either way. I can combine lines that are unlikely to fail, like these here.